modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContextTest.java

```diff
@@ -874,40 +886,37 @@ private void _setUpLanguageUtil() {
 	}
 
 	private void _setUpLocaleUtil() {
-		mockStatic(LocaleUtil.class);
+		LocaleUtil localeUtil = ReflectionTestUtil.getFieldValue(
+			LocaleUtil.class, "_localeUtil");
 
-		when(
-			LocaleUtil.fromLanguageId(_DEFAULT_LANGUAGE_ID)
-		).thenReturn(
-			LocaleUtil.SPAIN
-		);
+		Map<String, Locale> locales = ReflectionTestUtil.getFieldValue(
+			localeUtil, "_locales");
 
-		when(
-			LocaleUtil.fromLanguageId("pt_BR")
-		).thenReturn(
-			LocaleUtil.BRAZIL
-		);
+		locales.clear();
+
+		locales.put(_DEFAULT_LANGUAGE_ID, LocaleUtil.SPAIN);
+		locales.put("pt_BR", LocaleUtil.BRAZIL);
 	}
 
 	private void _setUpPortalUtil() {
 		PortalUtil portalUtil = new PortalUtil();
 
-		portalUtil.setPortal(mock(Portal.class));
+		portalUtil.setPortal(Mockito.mock(Portal.class));
 
-		when(
+		Mockito.when(
 			PortalUtil.getHttpServletRequest(Matchers.any(RenderRequest.class))
```

Comment:
@JanisZhang Please SF this to 

```
Portal portal = Mockito.mock(Portal.class);

portalUtil.setPortal(portal);

Mockito.when(
    portal.getHttpServletRequest(......)
)...
```

This is more clear that you are mocking `Portal`, not mocking the static `PortalUtil` methods.

Reply:
@dantewang sure